src/components/summary-card.js

```diff
@@ -10,63 +11,61 @@ const SummaryCard = (props) => {
 
   return (
     <div style={{ display: 'block', margin: '20px' }}>
-      <Card fluid raised>
-        <Card.Content>
-          <Grid>
-            <Grid.Row className='my-grid-top'>
-              <Label color= 'blue' style={{ margin: '0 0 8px 10px' }}>Level One</Label>
-              <Label color= 'blue' style={{ margin: '0 0 8px 10px' }}>Introductory Course</Label>
-              <Label color= 'blue' style={{ margin: '0 0 8px 10px' }}>something really long</Label>
-            </Grid.Row>
-            <Grid.Row className='my-grid-row'>
-              <div>
-                <Card.Header as='h2'>{summaryTitle}</Card.Header>
-                <a href={summaryLink} rel="noreferrer" target="_blank">
+      <Grid>
+        <Grid.Row className='my-grid-top'>
+          <div className='my-course-page-tags'>
+            <LabelExampleIcon code="Level 1"/>
+          </div>
+        </Grid.Row>
+        <Grid.Row className='my-grid-row'>
+          <div>
+            <Header as='h2'>{summaryTitle}</Header>
+            <a href={summaryLink} rel="noreferrer" target="_blank">
                     COMP1511 Handbook
-                </a> <br/>
-                <Rating icon='star' defaultRating={overallRating} maxRating={5} disabled/>
-                <Card.Meta>{numReviews} Reviews</Card.Meta>
-              </div>
-            </Grid.Row>
-            <Grid.Row className='my-grid-row'>
-              <div style={{ width: '100%' }}>
-                <span>
+            </a> <br/>
```

Comment:
better to set a margin (for example setting margin-bottom for this <a> tag) rather than inserting <br/>. this is because when youcopy paste a card it wil lcopy the <br/> too